app/models/sp_return_log.rb

```diff
@@ -6,5 +6,9 @@ class SpReturnLog < ApplicationRecord
   belongs_to :service_provider,
              foreign_key: 'issuer',
              primary_key: 'issuer'
+  belongs_to :profile_requested_service_provider,
+             class_name: 'ServiceProvider',
+             foreign_key: 'profile_requested_issuer',
+             primary_key: 'issuer'
```

Comment:
this adds an association that mirrors the one on `Profiles` table:

https://github.com/18F/identity-idp/blob/5c758fd167df84170312ec58ab8daba842d4c730/app/models/profile.rb#L6-L10

It means we can do:
```ruby
sp_return_log.profile_requested_issuer
# => (String)
sp_return_log,profile_requested_service_provider
# => (ServiceProvider)
```